### DISCUSSION TITLE
Case sensitivity of provisioner name?

### DISCUSSION THREAD

**Discussion by strarsis** [smallstep/certificates#686] [Q&A]
Certbot uses `acme` as provisioner name. However, when `step-ca` is configured to use `ACME` as provisioner name, it will not match requests from Certbot for `acme` as it compares the name case-sensitively, hence it will find now provisioner and fail to handle the certbot request.

Is case-sensitivity of the provisioner name something that is defined in the ACME RFC (can't find the word "provisioner" there though)? Or an assumed *nix-like behaviour?

**Comment by tashian** [comment]
@strarsis the provisioner name, from the perspective of an ACME client, is just part of the ACME server URL. And since URLs are case sensitive, it makes sense that a provisioner name would also be case sensitive.`step-ca` uses `/acme/<provisioner_name>` for its ACME endpoints. With certbot you can provide the server URL with `--server`, eg.`--server https://ca.internal/acme/ACME/directory`

Hope this helps.
